src/components/search/search.d.ts

```diff
@@ -40,6 +40,8 @@ export interface SearchProps extends MarginProps {
   variant?: string;
   /** A callback to retrieve the input reference */
   inputRef?: React.RefObject<HTMLInputElement>;
+  /** Input tabindex */
+  tabIndex?: number | string;
```

Comment:
`tabIndex` can only be a number I believe: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1349b640d4d07f40aa7c1c6931f18e3fbf667f3a/types/react/index.d.ts#L1764